.rubocop.yml

```diff
@@ -15,23 +15,24 @@ Layout/EmptyLineBetweenDefs:
 Layout/HeredocIndentation:
   Enabled: false
 Layout/LineLength:
+  Exclude:
+    - spec/**/*
   IgnoredPatterns:
-    - "^[ ]*#.+$"
-    - "^[ ]*describe.+$"
-    - "^[ ]*context.+$"
-    - "^[ ]*shared_context.+$"
-    - "^[ ]*shared_examples_for.+$"
-    - "^[ ]*it.+$"
-    - "^[ ]*'.+?' => '.+?',?$"
-    - "^[ ]*\".+?\" => \".+?\",?$"
+    - '^[ ]*#.+$'
+    - '^[ ]*''.+?'' => ''.+?'',?$'
+    - '^[ ]*".+?" => ".+?",?$'
+  Max: 120
```

Comment:
Sure, we can keep 80, but i changed to 120 thinking that 80 was because of rubocop's default that time, which is not changed to 120. So should we stay on 80 then?

Reply:
Yeah, I think it makes sense to stay on 80.